docs/docs/internal/governance--how-to-fast-track.md

```diff
@@ -0,0 +1,213 @@
+  - `proposalLengthInVoting`: How many blocks the proposal will stay in referenda.
+  - `proposalDelayUntilVoting`: How long the proposal will be delayed until it goes into referenda.
+- `lengthBound`: a value higher or equal to the length of the SCALE encoded proposal in bytes.[^1]
+
+#### Call example
+```typescript
+const {
+	data: [
+		resultAccount, resultProposalIndex, resultProposalHash, resultThreshold
+	]
+} = await sendAndWaitForSuccess(
+	api,
+	technicalCouncilMembers[0],
+	api.events.technicalCommittee.Proposed.is,
+	api.tx.technicalCommittee.propose(
+		threshold,
+		api.tx.democracy.fastTrack(
+			hashOfProposalToPropose, 
+			proposalLengthInVoting, 
+			proposalDelayUntilVoting
+		),
+		lengthBound
+	)
+);
+```
+
+
+### 5. Technical committee votes on proposal
+Now the members of the technical committee can vote on the newly created internal proposal.
+
+![5. Technical Committee: Proposal Voting](./governance--how-to-fast-track--images/5-technical-committee--proposal-voting.png)
+
+#### Parameters
+- `technicalCommitteeProposalHash`: Hash of the technical committee proposal. Received during #4 as `resultProposalHash`.
+- `technicalCommitteeProposalIndex`: Index number of the technical committee proposal. Received during #4 as `resultProposalIndex`.
+- `voteChoice`: true / false depending on the desired outcome of the technical committee proposal.
+
+#### Call example
+```typescript
+api.tx.technicalCommittee.vote(
+	technicalCommitteeProposalHash,
+	technicalCommitteeProposalIndex,
+	true
+)
+```
+
+
+### 6. Technical committee closes the proposal
+As soon as the voting threshold is reached, any technical committee member can close the proposal and execute it.
+
+![6. Technical Committee: Proposal Closing](./governance--how-to-fast-track--images/6-technical-commitee--proposal-closing.png)
+
+#### Parameters
+- `technicalCommitteeProposalHash`: Hash of the technical committee proposal. Received during #4 as "resultProposalHash".
+- `technicalCommitteeProposalIndex`: Index number of the technical committee proposal. Received during #4 as "resultProposalIndex".
+- `weightBound`: the maximum amount of Substrate weight the proposal can use. If the proposal call uses more, the call will revert.[^1]
+- `lengthBound`: a value higher or equal to the length of the SCALE encoded proposal in bytes.[^1]
+
+#### Call example
+```typescript
+api.tx.technicalCommittee.close(
+	technicalCommitteeProposalHash, 
+	technicalCommitteeProposalIndex, 
+	weightBound, 
+	lengthBound
+)
+```
+
+
+### Fast Tracking was successful
+Now the desired proposal will be in voting and all users can vote for their desired outcome.
+This will be the case until it has reached the defined `proposalLengthInVoting` during step #4,
+ignoring the default time a proposal stays in referenda.
+
+The only remaining task at this point is to submit the preimage for the proposal.
+For an introduction into submitting pre-images, please take a look into the proposal guide.
+
```

Comment:
Can you add a sentence here of the minimum time required to execute a proposal from first proposal to execution in Picasso?

Reply:
Thanks for the reminder. 👍 
Already figured out today that the fast tracking enactment time can also be freely defined. E.g. no need to wait at all. But I forgot to add it. Taking care off it now.